instaclone/src/components/SearchBar.js

```diff
@@ -1,16 +1,17 @@
 import React from "react"
 import "./head.css"
-
+import {SearchBarDiv,IconDiv,Horizontal} from "./../reusableComp/styles."
 const SearchBar = (props)=>{
```

Comment:
You know me, I'm not a big fan of having a style variable at the top of your component's file. But good used of styled components.